### DISCUSSION TITLE
Differents values with same parameters

### DISCUSSION THREAD

**Discussion by leofuy** [Ifsttar/I-Simpa#365] [General]
Hi,
Im running isimpa with same parameters but it give me differentes values fot RT15 when i run it several times. Even there are some high values in the second column.
What do you think it could be happening?

<TABLE FRAME=VOID CELLSPACING=0 COLS=3 RULES=NONE BORDER=0>
	<COLGROUP><COL WIDTH=86><COL WIDTH=86><COL WIDTH=86></COLGROUP>
	<TBODY>
		<TR>
			<TD WIDTH=86 HEIGHT=22 ALIGN=CENTER>RT-15 (s)</TD>
			<TD WIDTH=86 ALIGN=LEFT>RT-15 (s)</TD>
			<TD WIDTH=86 ALIGN=LEFT>RT-15 (s)</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,529124" SDNUM="14346;0;0,00">1,53</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,455113" SDNUM="14346;0;0,00">1,46</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,484952" SDNUM="14346;0;0,00">1,48</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,542333" SDNUM="14346;0;0,00">1,54</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,409004" SDNUM="14346;0;0,00">1,41</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,361835" SDNUM="14346;0;0,00">1,36</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,8417234" SDNUM="14346;0;0,00">0,84</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,6102098" SDNUM="14346;0;0,00">0,61</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,8578894" SDNUM="14346;0;0,00">0,86</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,3826655" SDNUM="14346;0;0,00">0,38</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFCC00" SDVAL="3,920249" SDNUM="14346;0;0,00">3,92</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,6355923" SDNUM="14346;0;0,00">0,64</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,8743567" SDNUM="14346;0;0,00">0,87</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,6515647" SDNUM="14346;0;0,00">0,65</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,6485892" SDNUM="14346;0;0,00">0,65</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,880627" SDNUM="14346;0;0,00">0,88</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="2,690289" SDNUM="14346;0;0,00">2,69</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,9476968" SDNUM="14346;0;0,00">0,95</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,058319" SDNUM="14346;0;0,00">1,06</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="2,34112" SDNUM="14346;0;0,00">2,34</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,045672" SDNUM="14346;0;0,00">1,05</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,008472" SDNUM="14346;0;0,00">1,01</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="1,789405" SDNUM="14346;0;0,00">1,79</TD>
			<TD ALIGN=CENTER BGCOLOR="#FFFF00" SDVAL="0,9894257" SDNUM="14346;0;0,00">0,99</TD>
		</TR>
	</TBODY>
</TABLE>
</BODY>

</HTML>

Thanks for your time
Leonardo

**Comment by nicolas-f** [comment]
Hi,

SPPS is based on random emission of particles, so if you not have enough particle for your simulation you have varying result for each different random seed.

**Reply by nicolas-f** [reply]
https://i-simpa-wiki.readthedocs.io/fr/latest/project_window.html#spps-calculation-parameters

**Reply by leofuy** [reply]
Hi nicolás,
Thanks for your answer.
Is there some kind of formule that relate the number of particles and the volume of the model?
Leonardo

**Reply by nicolas-f** [reply]
No. But it is easy to spot it when you visualize the sound level of a receiver over time. Noise level should become more and more diffuse over time, but if you do not have enough particle you will see positive and negative peaks after some simulation time.

**Reply by leofuy** [reply]
This is what you talk about?
I have 250.000 particles.
The volume is 1316.
This is the sound level in one point:
<HTML>

<BODY TEXT="#000000">
<TABLE FRAME=VOID CELLSPACING=0 COLS=21 RULES=NONE BORDER=0>
	<COLGROUP><COL WIDTH=62><COL WIDTH=54><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=61><COL WIDTH=68></COLGROUP>
	<TBODY>
		<TR>
			<TD WIDTH=62 HEIGHT=17 ALIGN=LEFT><BR></TD>
			<TD WIDTH=54 ALIGN=LEFT>5.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>10.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>15.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>20.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>25.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>30.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>35.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>40.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>45.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>50.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>55.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>60.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>65.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>70.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>75.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>80.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>85.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>90.0 ms</TD>
			<TD WIDTH=61 ALIGN=LEFT>95.0 ms</TD>
			<TD WIDTH=68 ALIGN=LEFT>100.0 ms</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>50 Hz</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>75.1</TD>
			<TD ALIGN=LEFT>87.1</TD>
			<TD ALIGN=LEFT>91.3</TD>
			<TD ALIGN=LEFT>93.1</TD>
			<TD ALIGN=LEFT>93.9</TD>
			<TD ALIGN=LEFT>92.4</TD>
			<TD ALIGN=LEFT>91.4</TD>
			<TD ALIGN=LEFT>90.7</TD>
			<TD ALIGN=LEFT>91.3</TD>
			<TD ALIGN=LEFT>90.7</TD>
			<TD ALIGN=LEFT>89.6</TD>
			<TD ALIGN=LEFT>89.0</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>63 Hz</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>75.0</TD>
			<TD ALIGN=LEFT>86.5</TD>
			<TD ALIGN=LEFT>90.8</TD>
			<TD ALIGN=LEFT>91.9</TD>
			<TD ALIGN=LEFT>93.6</TD>
			<TD ALIGN=LEFT>92.3</TD>
			<TD ALIGN=LEFT>91.3</TD>
			<TD ALIGN=LEFT>90.6</TD>
			<TD ALIGN=LEFT>91.6</TD>
			<TD ALIGN=LEFT>90.4</TD>
			<TD ALIGN=LEFT>89.2</TD>
			<TD ALIGN=LEFT>88.9</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>125 Hz</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>77.0</TD>
			<TD ALIGN=LEFT>88.0</TD>
			<TD ALIGN=LEFT>89.5</TD>
			<TD ALIGN=LEFT>92.9</TD>
			<TD ALIGN=LEFT>93.7</TD>
			<TD ALIGN=LEFT>91.6</TD>
			<TD ALIGN=LEFT>91.5</TD>
			<TD ALIGN=LEFT>89.2</TD>
			<TD ALIGN=LEFT>91.5</TD>
			<TD ALIGN=LEFT>91.3</TD>
			<TD ALIGN=LEFT>89.2</TD>
			<TD ALIGN=LEFT>89.9</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>250 Hz</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>74.3</TD>
			<TD ALIGN=LEFT>87.7</TD>
			<TD ALIGN=LEFT>89.1</TD>
			<TD ALIGN=LEFT>91.3</TD>
			<TD ALIGN=LEFT>92.6</TD>
			<TD ALIGN=LEFT>89.5</TD>
			<TD ALIGN=LEFT>89.0</TD>
			<TD ALIGN=LEFT>88.0</TD>
			<TD ALIGN=LEFT>88.6</TD>
			<TD ALIGN=LEFT>88.4</TD>
			<TD ALIGN=LEFT>86.5</TD>
			<TD ALIGN=LEFT>86.1</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>500 Hz</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>76.3</TD>
			<TD ALIGN=LEFT>88.0</TD>
			<TD ALIGN=LEFT>90.4</TD>
			<TD ALIGN=LEFT>89.5</TD>
			<TD ALIGN=LEFT>91.1</TD>
			<TD ALIGN=LEFT>86.3</TD>
			<TD ALIGN=LEFT>85.9</TD>
			<TD ALIGN=LEFT>83.7</TD>
			<TD ALIGN=LEFT>86.1</TD>
			<TD ALIGN=LEFT>85.1</TD>
			<TD ALIGN=LEFT>78.6</TD>
			<TD ALIGN=LEFT>80.4</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>1000 Hz</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>77.0</TD>
			<TD ALIGN=LEFT>86.8</TD>
			<TD ALIGN=LEFT>89.5</TD>
			<TD ALIGN=LEFT>88.3</TD>
			<TD ALIGN=LEFT>89.3</TD>
			<TD ALIGN=LEFT>83.4</TD>
			<TD ALIGN=LEFT>85.2</TD>
			<TD ALIGN=LEFT>82.4</TD>
			<TD ALIGN=LEFT>84.7</TD>
			<TD ALIGN=LEFT>83.3</TD>
			<TD ALIGN=LEFT>63.4</TD>
			<TD ALIGN=LEFT>73.7</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>2000 Hz</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>74.2</TD>
			<TD ALIGN=LEFT>86.0</TD>
			<TD ALIGN=LEFT>89.6</TD>
			<TD ALIGN=LEFT>89.0</TD>
			<TD ALIGN=LEFT>89.7</TD>
			<TD ALIGN=LEFT>82.6</TD>
			<TD ALIGN=LEFT>85.2</TD>
			<TD ALIGN=LEFT>81.3</TD>
			<TD ALIGN=LEFT>84.4</TD>
			<TD ALIGN=LEFT>84.5</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>4000 Hz</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>75.3</TD>
			<TD ALIGN=LEFT>87.2</TD>
			<TD ALIGN=LEFT>89.1</TD>
			<TD ALIGN=LEFT>89.3</TD>
			<TD ALIGN=LEFT>88.7</TD>
			<TD ALIGN=LEFT>83.4</TD>
			<TD ALIGN=LEFT>85.6</TD>
			<TD ALIGN=LEFT>82.8</TD>
			<TD ALIGN=LEFT>85.1</TD>
			<TD ALIGN=LEFT>81.8</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>70.3</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>8000 Hz</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>77.8</TD>
			<TD ALIGN=LEFT>85.7</TD>
			<TD ALIGN=LEFT>88.3</TD>
			<TD ALIGN=LEFT>87.1</TD>
			<TD ALIGN=LEFT>86.1</TD>
			<TD ALIGN=LEFT>82.7</TD>
			<TD ALIGN=LEFT>83.1</TD>
			<TD ALIGN=LEFT>81.6</TD>
			<TD ALIGN=LEFT>81.1</TD>
			<TD ALIGN=LEFT>79.3</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>65.5</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=LEFT>Global</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>-inf</TD>
			<TD ALIGN=LEFT>85.5</TD>
			<TD ALIGN=LEFT>96.6</TD>
			<TD ALIGN=LEFT>99.4</TD>
			<TD ALIGN=LEFT>100.3</TD>
			<TD ALIGN=LEFT>101.2</TD>
			<TD ALIGN=LEFT>98.5</TD>
			<TD ALIGN=LEFT>98.2</TD>
			<TD ALIGN=LEFT>96.7</TD>
			<TD ALIGN=LEFT>98.0</TD>
			<TD ALIGN=LEFT>97.3</TD>
			<TD ALIGN=LEFT>94.9</TD>
			<TD ALIGN=LEFT>94.9</TD>
		</TR>
	</TBODY>
</TABLE>
</BODY>

</HTML>

[fachada.csv](https://github.com/Universite-Gustave-Eiffel/I-Simpa/files/12081096/fachada.csv)

**Reply by nicolas-f** [reply]
Yes you see jittering is starting as soon as 660ms here

![image](https://github.com/Universite-Gustave-Eiffel/I-Simpa/assets/1382241/eee8d62f-a150-49ef-b081-c74a56102bdf)

You should use "energetic" for spps method and see the result, increasing particle number if it is still jittering

**Reply by leofuy** [reply]
Estimado Nicolás,
Here i attach a new file.
It still have some errors with 850.000 particles and this config:
![image](https://github.com/Universite-Gustave-Eiffel/I-Simpa/assets/139467798/6676455f-fad1-4540-9e12-23e653d6b9da)

**Comment by nicolas-f** [comment]
Hi.

For RT-15 you only need 20 dB attenuation from the peak (-5 dB -> -20 dB):

https://i-simpa-wiki.readthedocs.io/fr/latest/room_acoustics_parameters.html#reverberation-time-t

"Limit value" is a parameter that will destroy sound particle after some attenuation is reached. Here it will destroy particle that lost more than 50 dB.
